### PR TITLE
add QPS and burst

### PR DESCRIPTION
**Pull request by carmark**
Signed-off-by: Lei Xue <vfs@live.com>

### CODE REVIEW

File: cmd/mpi-operator.v1/app/options/options.go [hunk 1]
```diff
@@ -70,4 +72,7 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&s.LockNamespace, "lock-namespace", "mpi-operator", "Set locked namespace name while enabling leader election.")
 
 	fs.BoolVar(&s.LauncherRunsWorkload, "launcher-runs-workloads", false, "Set launcher run the workload when launcher has GPU")
+
+	fs.IntVar(&s.QPS, "qps", 5, "QPS indicates the maximum QPS to the master from this client.")
+	fs.IntVar(&s.Burst, "burst", 10, "Maximum burst for throttle.")
```
Comment (terrytangyuan) [inline cmd/mpi-operator.v1/app/options/options.go:77]: Do they match the k8s default?

Comment (carmark) [reply cmd/mpi-operator.v1/app/options/options.go]: done.

**Pull request merged**